kb_text_shape.h

```diff
@@ -13739,12 +13739,12 @@ static kbts_class_sequence_rule *kbts_GetClassSequenceRule(kbts_class_sequence_r
   return Result;
 }
 
-static kbts_coverage *kbts_GetCoverage(kbts_sequence_context_3 *Context, kbts_un Index)
-{
-  kbts_u16 *Offsets = (kbts_u16 *)(Context + 1);
-  kbts_coverage *Result = KBTS_POINTER_OFFSET(kbts_coverage, Context, Offsets[Index]);
-  return Result;
-}
+// static kbts_coverage *kbts_GetCoverage(kbts_sequence_context_3 *Context, kbts_un Index)
+// {
+//   kbts_u16 *Offsets = (kbts_u16 *)(Context + 1);
+//   kbts_coverage *Result = KBTS_POINTER_OFFSET(kbts_coverage, Context, Offsets[Index]);
+//   return Result;
+// }
```

Comment:
Not sure about your preference here, but assuming they will be used at a later date?